sentry_wrapper/__init__.py

```diff
@@ -11,11 +11,47 @@
 __version__ = '2.4.0'
 
 
-def wrap(dist, group, name, sentry_dsn, timeout=None):
+class SentryConfig(object):
+    def __init__(
+        self,
+        dsn,
+        release=None,
+        environment=None,
+        server_name=None,
+        attach_stacktrace=None,
+        with_locals=None,
+    ):
+        self.dsn = dsn
```

Comment:
nit: Do you need to store these variables? Does `self._conf_values` is not enough?

Reply:
Does not hurt to have them, could be useful later.